src/networktables/NetworkTable.cpp

```diff
@@ -43,6 +42,8 @@ void NetworkTable::SetTeam(int team) {
   char tmp[30];
 #ifdef _MSC_VER
   sprintf_s(tmp, "roboRIO-%d-FRC.local\n", team);
+#elif defined(__ANDROID__)
+  snprintf(tmp, 30, "roboRIO-%d-FRC.local\n",team);
```

Comment:
It does look like the cstdio header is included with the various C++ implementations that come with the NDK. Android's default C++ support is *very* limited, and I'm actually kind of surprised everything else compiles. Thad, are you using one of the helper runtimes? If so, which one? It would be nice to see some documentation updates so others don't have to do the same learning process you've been having to.